docs/user-guide.md

```diff
@@ -0,0 +1,659 @@
+# Louketo Proxy
+
+Louketo provides a Go programming language adapter for use with
+OpenID Connect (OIDC) that supports both access tokens in a browser
+cookie or bearer tokens.
```

Comment:
+1 This statement is a bit confusing as it seems it's a Go adapter, not a proxy

Reply:
Just passing through, but mentioning "Go" this early leads newcomers to think, "Oh, this is the adapter you use if your client is written in Go," which is entirely incorrect. I know your sentence doesn't _say_ that. But this can easily get accidentally get mis-categorized in newcomers' brains as yet another language-specific adapter, when it's much more general purpose than that.

Reply:
Thanks for your input @charlieflowers, but it was already updated. If you find anything else that should change, please let me know.